src/foundry/client/pixi/layers/effects/weather-effects.d.ts

```diff
@@ -0,0 +1,167 @@
+export {};
+
+declare global {
+  /**
+   * A CanvasLayer for displaying visual effects like weather, transitions, flashes, or more
+   */
+  class WeatherEffects extends FullCanvasObjectMixin(CanvasLayer) {
+    /**
+     * @defaultValue `true`
+     */
+    override sortableChildren: boolean;
+
+    /**
+     * @defaultValue `"none"`
+     */
+    override eventMode: PIXI.EventMode;
+
+    /**
+     * Sorting values to deal with ties.
+     * @defaultValue `1000`
+     */
+    static PRIMARY_SORT_ORDER: number;
+
+    /**
+     * @defaultValue `foundry.utils.mergeObject(super.layerOptions, {name: "effects"})`
```

Comment:
Nit; formatting should follow Prettier's styling so `{ name: "effect" }`. Yeah it's not that important.